src/hocon_schema.erl

```diff
@@ -71,6 +71,26 @@ translations(#{translations := Trs}) -> Trs.
 translation(Mod, Name) when is_atom(Mod) -> Mod:translation(Name);
 translation(#{translation := Tr}, Name) -> Tr(Name).
 
+generate(Schema, RichMap) ->
```

Comment:
Please add a comment about the purpose of this function and a typespec

Reply:
ty added